book_maker/translator/qwen_translator.py

```
@@ -0,0 +1,262 @@
+        "estonian": "Estonian",
+        # Add more mappings as needed
+    }
+
+    def __init__(
+        self,
+        key,
+        language,
+        model="qwen-mt-turbo",
+        source_lang="auto",
+        api_base=None,
+        prompt_template=None,  # Not used for translation models
```

Ruff (0.14.5)

Unused method argument: `prompt_template`

(ARG002)